### ISSUE TITLE
SGLD Variance Noise

### ISSUE THREAD

**Issue by emilemathieu** [blei-lab/edward#722]
Hello all,

I wonder whether there is a mistake in the SGLD sampler; the variance of the injectes noise should be `learning_rate` and the `Normal.scale` parameter corresponds to the std. Thus in

```python
normal = Normal(loc=tf.zeros(event_shape), scale=learning_rate * tf.ones(event_shape))
```

Should not it be ?
```python
normal = Normal(loc=tf.zeros(event_shape), scale=tf.sqrt(learning_rate) * tf.ones(event_shape))
```

It seems to be correct in SGHMC:  
```python
normal = Normal(loc=tf.zeros(event_shape), scale=(tf.sqrt(learning_rate * friction) * tf.ones(event_shape)))
```

Best, Emile

**Comment by dustinvtran** [comment]
Good catch! You are correct. PR is welcome.

**Comment by emilemathieu** [comment]
@dustinvtran : PR can be found here https://github.com/blei-lab/edward/pull/731

**Comment by dustinvtran** [comment]
Fixed in #731.
